rules.bolt

```diff
@@ -33,6 +33,7 @@ path /ui {
 
 path /projects {
     read() { true }
+    write() { isAdmin() }
```

Comment:
This is wrong. This can make admin add projects in any structure. We need proper structuring for projects. See news, contribution, profile, etc for how it is done